cloud_experts_tutorials/cloud-experts-rosa-with-hcp-private-offer-acceptance-and-sharing.adoc

```diff
@@ -0,0 +1,207 @@
++
+[NOTE]
+====
+{hcp-title} private offer cannot be shared with AWS linked accounts through the AWS License Manager.
+====
+
+. Add any users that you want to deploy ROSA clusters. Check link:https://access.redhat.com/customer-service-users[this user management FAQ] for more details about Red{nbsp}Hat account user management tasks. 
+
+. Verify that the already logged in Red{nbsp}Hat account includes all users that are meant to be ROSA cluster deployers benefiting from the accepted private offer.
+
+. Verify that the Red{nbsp}Hat account number and the AWS account ID are the desired accounts that are to be linked. This linking is unique and a Red{nbsp}Hat account can be connected only with a single AWS (billing) account.
++
+.AWS and Red{nbsp}Hat accounts connection
++
+image::rosa-aws-and-red-hat-accounts-connection.png[]
+
+. If you want to link the AWS account with another Red{nbsp}Hat account than is shown on this page in Figure 9, log out from the Red{nbsp}Hat Hybrid Cloud Console before connecting the accounts and repeat the step of setting the account by returning to the private offer URL that is already accepted.
++
+An AWS account can be connected with a single Red{nbsp}Hat account only. Once Red{nbsp}Hat and AWS accounts are connected, this cannot be changed by the user. If a change is needed, the user must create a support ticket.
++
+The account linking and initial cluster deployment steps are described in more detail in xref:../cloud_experts_tutorials/cloud-experts-rosa-hcp-activation-and-account-linking-tutorial.adoc#aws-and-redhat-account-and-subscription-linking[AWS and Red{nbsp}Hat account and subscription linking].
+
+. Agree to the terms and conditions and then click *Connect accounts*.
+
+== AWS billing account selection
+
+* When deploying {hcp-title} clusters, verify that end users select the AWS billing account that accepted the private offer. 
+
+* When using the web interface for deploying {hcp-title}, the _Associated AWS infrastructure account_ is typically set to the AWS account ID used by the administrator of the cluster that is being created. 
+** This can be the same AWS account as the billing AWS account. 
+** AWS resources are deployed into this account and all the billing associated with those resources are processed accordingly.
++
+.Infrastructure and billing AWS account selection during {hcp-title} cluster deployment
++
+image::rosa-infrastructure-and-billing-aws-account-selection-during-rosa-with-hcp-cluster-deployment.png[]
++
+** The drop-down for the AWS billing account on the screenshot above should be set to the AWS account that accepted the private offer, providing the purchased quota is intended to be used by the cluster that is being created. If different AWS accounts are selected in the infrastructure and billing _roles_, the blue informative note visible in Figure 10 is shown.
```

Comment:
Other instances of "drop down" were without the hyphen. I think they need to be consistent.